tests/epyccel/test_functionals.py

```diff
@@ -1,43 +1,99 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring/
+import pytest
 from numpy.random import randint
 from numpy import equal
 
 from pyccel.epyccel import epyccel
 from modules import functionals
 
-def compare_epyccel(f, *args):
-    f2 = epyccel(f)
-    out1 = f(*args)
-    out2 = f2(*args)
-    assert equal(out1, out2).all()
+#==============================================================================
+class epyccel_test:
+    """
+    Class to pyccelize module then compare different results
+    This avoids the need to pyccelize the file multiple times
+    or write the arguments multiple times
+    """
```

Comment:
Nice idea!

This should save time when the same function is called multiple times with different arguments. But I don't see that happening in this file, maybe you wanted to add more tests?

By the way, I think that in the docstring you meant to write "Class to pyccelize function", not "module". Is that right?